src/components/molecules/Monaco/codeIntel.ts

```diff
@@ -291,7 +328,70 @@ export async function applyForResource(
     }
   });
 
-  return {newDecorations, newDisposables};
+  return {decorations: newDecorations, disposables: newDisposables};
+}
+
+function applyPolicyIntel(resource: K8sResource): {
+  decorations: monaco.editor.IModelDeltaDecoration[];
+  markers: monaco.editor.IMarkerData[];
+} {
+  const issues = resource.issues?.errors ?? [];
+
+  const glyphs = issues.map(issue => {
```

Comment:
More a functional rather than technical question. I think we do not want to show the glyphs in the gutter. I checked how VSC does it and they only do underlining on errors. We need to investigate the UX of many errors though as including descriptions to it will give a popup over your whole screen. (Most language errors have a single error so it's different).

Reply:
Keeping both for now since selecting parent node for missing error will obfuscate error of a more detailled node otherwise.